conans/client/command.py

```diff
@@ -327,30 +190,13 @@ def config(self, *args):
 
         args = parser.parse_args(*args)
 
-        config_parser = ConanClientConfigParser(self._client_cache.conan_conf_path)
-        if args.subcommand == "set":
```

Comment:
This looks like as argument parsing, doesn't it? Maybe the api should be like ``config_set(key, value)``, ``config_get()`` instead of ``config("set", "key="value")``?